.rubocop.yml

```diff
@@ -217,6 +217,7 @@ Layout/SpaceAroundOperators:
     - '**/spec/actions_specs/xcodebuild_spec.rb'
 
 AllCops:
+  TargetRubyVersion: 2.4
```

Comment:
Is this true?

We are still supporting System Ruby (2.0.0).
I think this setting isn't suitable.